app/assets/stylesheets/application.scss

```diff
@@ -33,16 +37,16 @@ body {
 }
 
 main {
-	margin-top: 3em;
-  padding: 0;
+	margin-top: 0em;
+    padding: 0;
 }
 
 * {
 	/*max-width: 100%;*/
 	box-sizing: border-box;
 }
 
-a {
+a:not(.filterlink){
```

Comment:
This is messing up the CSS for all buttons on the app. Fix this please-- and be extra careful when you're messing with application.scss